BREAKINGCHANGES_dev.md

```diff
@@ -0,0 +1,3 @@
+# Configutation
+
+1. The `enable_winget` configuration entry in the `windows` section has been removed because it will not cause any issues and will be enabled by default.
```

Comment:
Please [wrap in 80](https://github.com/topgrade-rs/topgrade/blob/main/CONTRIBUTING.md#breaking-changes)

Reply:
like this